wandb/sdk/wandb_run.py

```diff
@@ -3196,7 +3244,7 @@ def link_model(
             run.link_model(
                 path="/local/directory",
                 registered_model_name="my_reg_model",
```

Comment:
can we use a registered model name representative of an ML task, such as Review Summarization, or Object Detection, etc?

Reply:
Same opinion as above.